packages/grafana-ui/src/components/Forms/InlineField.story.tsx

```diff
@@ -11,40 +11,74 @@ import mdx from './InlineField.mdx';
 const meta: ComponentMeta<typeof InlineField> = {
   title: 'Forms/InlineField',
   component: InlineField,
+  argTypes: {
+    label: { control: { type: 'text' } },
+    labelWidth: { control: { type: 'number', min: 12 } },
```

Comment:
This `min` is not very enforced, just if you are using the arrows to go up/down in the number but you can also type any other value in the box... So I'd say we don't need to add it, might be interesting for some people to see what it looks like if the width is very small.